streaming/flink/helm-values/metrics-server-values.yaml

```diff
@@ -0,0 +1,52 @@
+# HA config for metrics-server
+image:
+  repository: k8s.gcr.io/metrics-server/metrics-server
```

Comment:
Use `registry.k8s.io` instead. See https://aws.amazon.com/blogs/containers/changes-to-the-kubernetes-container-image-registry/

Reply:
Thanks for this. Did a find and replace on this repo